catalog/ui/pom.xml

```diff
@@ -30,6 +30,7 @@
         <module>search-ui</module>
         <module>catalog-ui-search</module>
         <module>search-ui-app</module>
+        <module>catalog-ui-enumeration</module>
```

Comment:
If `catalog-ui-search` is dependent on this module, Then this should be above it in this module listing (since on **serial** builds the module build order is determined from this module list). You should be able to break it in it's current state by cleaning everything and doing a serial build (You might have to have cleaned .m2 as well if you want to repro the failure)